src/sensors.py

```diff
@@ -7,199 +7,166 @@
 
 
 @tornado.gen.coroutine
-def _sensor_cb_flru(instr, sensor, executor):
+def _sensor_cb_flru(instr, sensor, executor, host_name):
```

Comment:
Should `host_name` not be something like 'fpga' or 'f_host'/'x_host'? You are passing the function an FPGA host object, not the host name, so this parameter name is a bit confusing.